src/renderer/components/NewProjectModal/RunTranscriptionView.tsx

```diff
@@ -1,47 +1,35 @@
-import { Box, styled, Typography, colors, Button } from '@mui/material';
+import { Box, styled, Stack, Typography, Button } from '@mui/material';
 import { useDispatch, useSelector } from 'react-redux';
+import colors from 'renderer/colors';
+import CloseIcon from '@mui/icons-material/Close';
+import IconButton from '@mui/material/IconButton';
 import { useCallback, useEffect, useState } from 'react';
+import { extractFileNameWithExtension } from 'renderer/util';
 import { transcriptionCreated } from '../../store/actions';
 import { ApplicationStore } from '../../store/helpers';
-import { Transcription } from '../../../sharedTypes';
+import { Transcription, AsyncState } from '../../../sharedTypes';
+import MediaDisplayTranscribeProgress from '../MediaDisplayTranscribeProgress';
 
-const ButtonContainer = styled(Box)`
-  display: flex;
-  justify-content: right;
+const CustomStack = styled(Stack)`
   width: 100%;
 `;
 
-const CustomButtonBase = styled(Button)`
-  border-radius: 0;
-  font-weight: bold;
-  color: ${colors.grey[900]};
-  text-transform: none;
-  margin: 20px 10px;
-  padding: 10px;
+const CustomColumnStack = styled(CustomStack)`
+  flex-direction: column;
 `;
 
-const CancelButton = styled(CustomButtonBase)`
-  background: ${colors.grey[400]};
-
-  &:hover {
-    background: ${colors.grey[600]};
-  }
+const CustomRowStack = styled(CustomStack)`
+  flex-direction: row;
+  align-items: center;
 `;
 
-const ActionButton = styled(CustomButtonBase)`
-  background: ${colors.lightBlue[500]};
-
-  &:hover {
-    background: ${colors.lightBlue[700]};
-  }
+const Container = styled(Box)`
+  background-color: ${colors.grey[700]};
 `;
 
-enum AsyncState {
-  READY = 'READY',
-  LOADING = 'LOADING',
-  DONE = 'DONE',
-  ERROR = 'ERROR',
-}
+const CustomButton = styled(Button)`
+  filter: drop-shadow(0px 8px 16px rgba(0, 0, 0, 0.8));
```

Comment:
@hhen0005 are we able to add drop shadow to the default button styling?

Reply:
Sent you a msg to create ticket 😃